app/Http/Controllers/OrderController.php

```
@@ -56,6 +57,7 @@
         $validated = $request->validated();
 
         $order = Order::create(array_merge($validated, [
+            'uuid' => Str::uuid()->toString(),
             'created_by' => $request->user()->id,
             'updated_by' => $request->user()->id
         ]));
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Order' in method 'store'. (Clean Code Rules)

(StaticAccess)